readthedocs/templates/projects/webhook_exchange_detail.html

```diff
@@ -0,0 +1,39 @@
+{% extends "projects/project_edit_base.html" %}
```

Comment:
this is mostly copy/paste from https://github.com/readthedocs/readthedocs.org/blob/webhooks/readthedocs/templates/projects/integration_exchange_detail.html

Reply:
Is there a way to share the same file instead of copy/paste?

Reply:
The body could be moved to another file. But I do see this having more options (like retry the webhook), so don't know.